buildlib/tools/builds.sh

```diff
@@ -427,8 +431,9 @@ do_task() {
 az_init_modules
 prepare_build
 
-[ "${long_test}" = "yes" ] && prog=5 || prog=12
+[ "${long_test}" = "yes" ] && prog=5 || prog=10
```

Comment:
maybe we could use array of commands, and set `prog` to 100/array_length:
- `tests=('check_config_h', 'check_inst_headers', ...)` + long runs and then:
- `for test in "${tests[@]}"; do`